deployer/constants.py

```diff
@@ -6,21 +6,30 @@
 DEFAULT_LOG_LEVEL = "INFO"
 
 DEFAULT_SCHEDULER_TIMEZONE = "Europe/Paris"
-DEFAULT_LOCAL_PACKAGE_PATH = Path("vertex/pipelines/compiled_pipelines")
 DEFAULT_TAGS = None
 
 TEMP_LOCAL_PACKAGE_PATH = ".vertex-deployer-temp"
 
-PIPELINE_MINIMAL_TEMPLATE = Path(TEMPLATES_PATH / "pipeline_minimal.py.jinja")
-PYTHON_CONFIG_TEMPLATE = Path(TEMPLATES_PATH / "python_config.py")
-JSON_CONFIG_TEMPLATE = Path(TEMPLATES_PATH / "json_config.json")
-TOML_CONFIG_TEMPLATE = Path(TEMPLATES_PATH / "toml_config.toml")
-DOCKERFILE_TEMPLATE = Path(TEMPLATES_PATH / "Dockerfile.jinja")
-CLOUDBUILD_LOCAL_TEMPLATE = Path(TEMPLATES_PATH / "cloudbuild_local.yaml.jinja")
-BUILD_BASE_IMAGE_TEMPLATE = Path(TEMPLATES_PATH / "build_base_image.sh.jinja")
+PIPELINE_MINIMAL_TEMPLATE = Path(TEMPLATES_PATH / "pipelines/pipeline_minimal.py.jinja")
+PYTHON_CONFIG_TEMPLATE = Path(TEMPLATES_PATH / "configs/python_config.py.jinja")
+JSON_CONFIG_TEMPLATE = Path(TEMPLATES_PATH / "configs/json_config.json.jinja")
+TOML_CONFIG_TEMPLATE = Path(TEMPLATES_PATH / "configs/toml_config.toml.jinja")
 
+DUMMY_COMPONENT_TEMPLATE = Path(TEMPLATES_PATH / "components/dummy_component.py.jinja")
 DEPLOYER_ENV_TEMPLATE = Path(TEMPLATES_PATH / "deployer.env.jinja")
-DEPLOYER_REQUIREMENTS_TEMPLATE = Path(TEMPLATES_PATH / "deployer-requirements.txt.jinja")
+REQUIREMENTS_VERTEX_TEMPLATE = Path(TEMPLATES_PATH / "requirements-vertex.txt.jinja")
+DOCKERFILE_TEMPLATE = Path(TEMPLATES_PATH / "deployment/Dockerfile.jinja")
+CLOUDBUILD_LOCAL_TEMPLATE = Path(TEMPLATES_PATH / "deployment/cloudbuild_local.yaml.jinja")
+BUILD_BASE_IMAGE_TEMPLATE = Path(TEMPLATES_PATH / "deployment/build_base_image.sh.jinja")
+
+TEMPLATES_DEFAULT_STRUCTURE = [
+    DUMMY_COMPONENT_TEMPLATE,
+    DEPLOYER_ENV_TEMPLATE,
+    REQUIREMENTS_VERTEX_TEMPLATE,
+    DOCKERFILE_TEMPLATE,
+    CLOUDBUILD_LOCAL_TEMPLATE,
+    BUILD_BASE_IMAGE_TEMPLATE,
+]
```

Comment:
Do you need these intermediate global constants or is the list enough / wouldn't it be better and easier if you had a dict ?